front-end/src/components/App.js

```diff
@@ -70,12 +55,13 @@ const App = () => {
   
   return (
     <div className="App">
-      <Topbar openCreateGraphModal={openCreateGraphModal} openUploadModal={openUploadModal} openHelpModal={openHelpModal}/>
+      <Topbar setModal={setModal}/>
       <header className="App-header">
         <div className="success">{successMessage.message}</div>
-        {showCreateGraphModal ? <CreateGraphModal setShowModal={setShowCreateGraphModal} setChartInformation={setChartInformation} setSuccessMessage={setSuccessMessage}/> : null}
-        {showUploadModal ? <UploadModal setShowUploadModal={setShowUploadModal} setSuccessMessage={setSuccessMessage}/> : null}
-        {showHelpModal ? <HelpModal setShowHelpModal={setShowHelpModal} /> : null}
+        {modal === 'Create' ? <CreateGraphModal setModal={setModal} setChartInformation={setChartInformation} setSuccessMessage={setSuccessMessage}/> : null}
```

Comment:
I like this, very nice

Reply:
thank you! (is that good to mark as resolved?)